client/components/Contacts.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState, useContext } from 'react';
+
+import { UserContext } from '../context/UserContext';
+import List from '@mui/material/List';
+import ListItemButton from '@mui/material/ListItemButton';
+import Divider from '@mui/material/Divider';
+import ListItemText from '@mui/material/ListItemText';
+import ListItemAvatar from '@mui/material/ListItemAvatar';
+import Avatar from '@mui/material/Avatar';
+import Typography from '@mui/material/Typography';
+
+const Contacts = () => {
+  const userContext = useContext(UserContext);
+  const { currentUserInfo, userContacts } = userContext;
+  const currentUser = currentUserInfo;
+  const userContext = useContext(UserContext);
+  const { currentUserInfo, userContacts } = userContext;
+  const [currentUserName, setCurrentUserName ] = useState(undefined)
+  const [ currentUserImage, setCurrentUserImage ] = useState(undefined)
+  const [ currentSelected, setCurrentSelected ] = useState(undefined)
+  const [selectedIndex, setSelectedIndex] = useState(undefined);

```

Comment:
why are you setting the initial state to undefined?

Reply:
i was kind of doing this weird combination of state and context then realized i dont really need the state for those things anyway i dont believe because of the context,  probably going to remove some of those